### ISSUE TITLE
TreeFeller reload fails to (re)load previously-loaded PAPI expansion

### ISSUE THREAD

**Issue by SlimeDog** [ThizThizzyDizzy/tree-feller#62]
Spigot 1.19.4-3703 or Paper 1.19.4-482
TreeFeller 1.21.4

```
>treefeller reload
[09:26:06] [Server thread/WARN]: [PlaceholderAPI] Failed to load expansion treefeller. Identifier is already in use.
[09:26:06] [Server thread/INFO]: Tree Feller reloaded!
```

**Comment by ThizThizzyDizzy** [comment]
should be harmless

**Comment by SlimeDog** [comment]
On Spigot/CraftBukkit, the stacking issue seems to happen more frequently on the cardinal points.

I also noticed that the trees are not dropping saplings. I have `spawn-saplings: 2`, but none are spawned. What am I missing?

**Comment by SlimeDog** [comment]
Of course. Just added to the list of (minor) annoyances. I check for ERROR and WARN messages on start-up. Making it clean is a nice feature.

**Comment by SlimeDog** [comment]
Confirmed fixed with TreeFeller 1.22.0. Thanks.
